packages/system/src/Location/ListView/api.ts

```diff
@@ -23,6 +28,56 @@ const locationsGuard = (locationsQuery: LocationsQuery) => {
   throw new Error(locationsQuery.locations.error.description);
 };
 
+const toInsertInput = (location: Location): InsertLocationInput => ({
+  id: location?.id,
+  name: location?.name,
+  code: location?.code,
+  onHold: location?.onHold,
+});
```

Comment:
these mappings seem redundant, but if we send *any* 'extra' fields through the API, there's an error and the request is rejected. So, for instance, if we have a `Location` object with an extra `__typename`, leftover from the request itself, then if we sent that back, the mutation would fail